src/main/java/com/champions/carsharingservice/controller/HealthController.java

```diff
@@ -9,6 +9,6 @@
 public class HealthController {
     @GetMapping("/hello")
     public String healthCheck() {
-        return "Hello world!";
```

Comment:
why?🤣

Reply:
I am friendly and like talking to world 😄